voxblox/src/integrator/tsdf_integrator.cc

```diff
@@ -53,13 +53,7 @@ TsdfIntegratorBase::TsdfIntegratorBase(const Config& config,
     : config_(config), layer_(layer) {
```

Comment:
nit: you could remove the initialization of `layer_(layer)`, it's done below in setLayer.